code/modules/robotics/bot/cambot.dm

```diff
@@ -278,11 +277,12 @@
 
 /obj/machinery/bot/cambot/proc/flash_blink(var/loops, var/delay)
 	set waitfor = 0
-	for (var/i=loops, i>0, i--)
-		add_simple_light("cambot", list(255,255,255,255 * (src.emagged ? 0.8 : 0.6)))
-		sleep(delay)
-		remove_simple_light("cambot")
-		sleep(delay)
+	SPAWN_DBG(0)
```

Comment:
there's already `set waitfor = 0` which is basically jsut fancy words for "this whole proc is in SPAWN_DBG(0)" so this is redundant.

Reply:
Aight, removed it. Didnt know that did that!